rental.rb

```diff
@@ -1,16 +1,14 @@
-require_relative './book'
-require_relative './person'
-
 class Rental
-  attr_accessor :date, :person, :book
+  attr_accessor :date
+  attr_reader :book, :person
 
-  def initialize(date, person, book)
+  def initialize(date, book, person, id)
     @date = date
 
+    @book = book
+
```

Comment:
- [ ] An error occurs in your app when trying to create a new rental as shown below 👇 Kindly take a look and try to fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/190870716-23838df7-460f-4edc-8260-d0abd700d4b9.png)